packages/eslint-plugin-pf-codemods/test/rules/v5/tableActionsColumn-removed-tooltipProp.js

```diff
@@ -0,0 +1,82 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/tableActionsColumn-removed-tooltipProp");
+
+ruleTester.run("tableActionsColumn-removed-tooltipProp", rule, {
+  valid: [
+    {
+      code: `<ActionsColumn items={[{tooltipProps: {content: "test"}}]} />`,
```

Comment:
Nit: shouldn't this valid test have the patternfly import?

Reply:
Whoops

Updated